internal/app/go-filecoin/node/block_propagate_test.go

```diff
@@ -213,7 +214,8 @@ func makeNodesBlockPropTests(t *testing.T, numNodes int) (address.Address, []*No
 	genTime := time.Unix(genUnixSeconds, 0)
 	fc := clock.NewFake(genTime)
 	blockTime := 30 * time.Second
-	c := clock.NewChainClockFromClock(1234567890, blockTime, fc)
+	propDelay := 5 * time.Second
```

Comment:
any reason to be inconsistent here?  if not the standard 6 would be a little simpler